Common/Basic comppenants/CatalogCategories.cs

```diff
@@ -0,0 +1,24 @@
+﻿using OpenQA.Selenium;
+using System.Linq;
+
+namespace Infrastructure
+{
+    public class CatalogCategories : ComppenantBase
+    {
```

Comment:
if you choose only one element in each variable, you should call it Woman, Dress, TShirt